rotkehlchen/externalapis/blockscout.py

```diff
@@ -120,19 +120,24 @@ def query_withdrawals(self, address: ChecksumEvmAddress) -> None:
         May raise:
         - RemoteError if blockscout query fails
         - DeserializationError if we can't decode the response
+        - KeyError
```

Comment:
as I mentioned in discord
```suggestion
        - KeyError if the blockscout response is missing an expected key
```